plugins/org.eclipse.n4js/src/org/eclipse/n4js/external/LibraryManager.java

```diff
@@ -308,6 +308,11 @@ private void collectDependencies(ProjectDescription description, Map<String, Str
 		for (ProjectDependency pDep : description.getProjectDependencies()) {
 			String name = pDep.getProjectName();
 			String version = NO_VERSION;
+
+			if (name.endsWith("-n4jsd")) {
```

Comment:
no tests, magic inlined strings, no documentation...
👍 ship it

Reply:
You know, I wanted to help as soon as possible. However, I added a comment.